dashboard/Dockerfile.broken

```
@@ -0,0 +1,169 @@
+FROM nginx:alpine as production
+
+# Install curl for health checks
+RUN apk add --no-cache curl
+
+# Copy custom nginx configuration
+COPY --from=builder /app/dist /usr/share/nginx/html
+
+# Create custom nginx configuration
+# Copy nginx configuration from file
+COPY nginx.conf /etc/nginx/conf.d/default.conf
+server {
```

Hadolint (2.12.0)

[error] unexpected 's'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)